### PR TITLE
Add a easy install instruction in readme

### PR DESCRIPTION
**Pull request by xfate123**
In order to teach beginner to install xgboost operator, I added a simple install instruction by using git clone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,15 @@ This repository contains the specification and implementation of `XGBoostJob` cu
...
+You can deploy the operator with default settings by running the following commands
+```
+git clone https://github.com/kubeflow/xgboost-operator
+cd xgboost-operator
+kubectl create -f manifests/xgboost-operator/base/cluster-role.yaml
+kubectl create -f manifests/xgboost-operator/base/crd.yaml
+kubectl create -f manifests/xgboost-operator/base/service-account.yaml
+kubectl create -f config/rbac/rbac_role_binding.yaml
+```
 XGBoost Operator is developed based on [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) and [Kubeflow Common](https://github.com/kubeflow/common).
```
Comment (merlintang) [inline README.md:27]: how to add the title here: ## Build XGboost Operator

**Comment by xfate123** [comment]
@googlebot I signed it!

**Comment by merlintang** [comment]
this is related to issue https://github.com/kubeflow/xgboost-operator/issues/57

**Comment by xfate123** [comment]
In order to teach beginner to install xgboost operator, I added a simple install instruction by using git clone

File: README.md (hunk 1 above)
Comment (xfate123) [reply README.md:28]: Title added

**Pull request merged**